tests/restore.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [dbg^|rel] [x86^|amd64^|arm] [-?]"
+    exit -8
+}
+
+
+source $(cd "$(dirname "$0")"; pwd -P)/testenv.sh
+
+if [ -z ${__CoreRT_BuildOS} ]; then
+    __CoreRT_BuildOS=Linux
+fi
+
+if [ -z ${__CoreRT_BuildArch} ]; then
+    echo "Set __CoreRT_BuildArch to x86/x64/arm/arm64"
+    exit -1
+fi
+
+if [ -z ${__CoreRT_BuildType} ]; then
+    echo "Set __CoreRT_BuildType to Debug or Release"
+    exit -1
+fi
+
+__build_os_lowcase=$(echo "${__CoreRT_BuildOS}" | tr '[:upper:]' '[:lower:]')
+if [ ${__build_os_lowcase} != "osx" ]; then
+    __BuildRid=ubuntu.14.04
+else
+    __BuildRid=osx.10.10
+fi
+__CoreRT_ToolchainPkg=toolchain.${__BuildRid}-${__CoreRT_BuildArch}.Microsoft.DotNet.ILCompiler.Development
+__CoreRT_ToolchainVer=1.0.2-prerelease-00001
+__CoreRT_AppDepSdkPkg=toolchain.${__BuildRid}-${__CoreRT_BuildArch}.Microsoft.DotNet.AppDep
+__CoreRT_AppDepSdkVer=1.0.2-prerelease-00002
+
+__ScriptDir=$(cd "$(dirname "$0")"; pwd -P)
+__BuildStr=${__CoreRT_BuildOS}.${__CoreRT_BuildArch}.${__CoreRT_BuildType}
+
+while test $# -gt 0
+    do
+        lowerI="$(echo $1 | awk '{print tolower($0)}')"
+        case $lowerI in
+        -?|-h|-help)
```

Comment:
Remove `-?` - it does not actually work, it is not Unix-friendly. @manu-silicon fixed it in one other script yesterday.